### PR TITLE
Added validation of verification, including mandatory parameters.

### LINKED ISSUE #37: Add verification experiment parameters validation
Contributes to #34

### PR DESCRIPTION
**Pull request by russmiles**
Closes #37

@Lawouach Asked for this review from you as this gives an indication of what I'm expecting as an experiment with a complete verification block, and how I'm ensuring I have what I need, as much as possible, before I go ahead and attempt to execute a verification.

Signed-off-by: Russ Miles <russ@russmiles.com>

### CODE REVIEW

**Comment by russmiles** [comment]
One question @Lawouach is should a verification _require_ a SSH with at least one entry. Otherwise, what are we validating? My original philosophy here was to be flexible and just do whatever gets handed to me, but I could be stricter...?

File: chaoscloud/verify/verification.py [hunk 1]
```diff
@@ -1,17 +1,55 @@
...
+    if not len(chaosiq_blocks) == 1:
+        raise InvalidVerification(
+                "a verification must have a single chaosiq extension block")
+
+    verification = chaosiq_blocks[0].get("verification")
+    if verification is None:
+        raise InvalidVerification(
+                "a verification must have a verification block")
+
+    id = verification.get("id")
```
Comment (Lawouach) [inline chaoscloud/verify/verification.py]: Just a FYI, `id` is also a function name in Python so your assignment here overwrites it in the scope of this function.

No need to change it, it's merely for your knowledge :)

**Comment by Lawouach** [comment]
Well, I think the validation of having a SSH should have been done when creating the verification on the console, don't you think? You can certainly have a check here but this would be a situtation of a hand crafted verification IMO :)

**Comment by russmiles** [comment]
I agree on the SSH @Lawouach . It’s not exactly a problem, just would make the verification odd and so I’ll not protect for it ... yet.

**Pull request merged**